### ISSUE TITLE
Will this project support MonoDevelop 7.0?

### ISSUE THREAD

**Issue by TW-YingjieOuyang** [hifi/monodevelop-justenoughvi#42]
As we know, the Visual Studio for Mac is launched. But for now I can't find any Vim extension for this new IDE. What only I can find after I search on Google is this plugin. I've tried install this manually, but seems it only supports MonoDevelop 6.0. I really hope that it will support MonoDevelop 7.0 and then I can use Vim on the new Visual Studio for Mac.  Thanks mate~

**Comment by hifi** [comment]
I don't have a Mac or Visual Studio for Mac so unless MD 7 is released for Linux or someone else who has that can do the port while keeping backwards compatibility with MD 6 it won't happen, sorry.

**Comment by TW-YingjieOuyang** [comment]
Oh, I see. Anyway, thanks mate!

**Comment by JordanTHarris** [comment]
Would updating to the Alpha channel of Xamarin Studio (or MonoDevelop) work? I know that's the only way you can have both VS for Mac and Xamarin Studio installed on a machine, so maybe it uses MD 7. I'll have to test this. Not sure if MonoDevelop is the same as I don't use that. Just an idea.

**Comment by hifi** [comment]
Actually I was mistaken when closing #54, the website is again outdated (http://www.monodevelop.com/download/) as 7.0 is available on MD flatpak.

I will try if I can push a release to 7.x channel soon.

**Comment by hifi** [comment]
Looking at the changes between 6.0 and 7.0 it seems this add-in would require a major refactor. I suggest looking at the XSVim add-in by @nosami, particularly the `7.0` branch at this point.

http://addins.monodevelop.com/Project/Index/267
https://github.com/nosami/XSVim

**Comment by nosami** [comment]
Yeah. Xsvim works with visual studio for Mac, but right now you need to
install manually via the mpack file on the github release page

**Comment by JordanTHarris** [comment]
Thank you for the suggestion. I'm trying XSVim out and it seems to work quite well. Somehow my google searches never led me to XSVim. It's a pain trying to use a text editor without Vim functionality. :)
